### PR TITLE
Bug/fix some a11y issues

### PR DESCRIPTION
**Pull request by nickytonline**
Resolves all of #118.

These are all fixes caught by the Axe browser extension ([Chrome](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd) | [Firefox](https://addons.mozilla.org/en-CA/firefox/addon/axe-devtools/)) used in conjunction with the [Web AIM contrast checker](https://webaim.org/resources/contrastchecker/).

## Before

image

## After

image

## Before

image

## After

image

Christopher Walken dancing with a hallway broom

Feel free to suggest better colours. These are the ones I came up with as soon as the Web AIM contrast checker gave me all passes.

### CODE REVIEW

File: src/components/form/PostAJobForm.js [hunk 1]
```diff
@@ -55,8 +55,9 @@ const PostAJobForm = ({
           companyDescription: Yup.string().required(
             'Please give a brief description of the company and culture.'
           ),
-          companyLogo: Yup.mixed().required('Please provide a .png format image of your company logo')
-            .test(file => file && file.type === 'image/png'),
+          companyLogo: Yup.mixed()
```
Comment (nickytonline) [inline src/components/form/PostAJobForm.js:58]: Prettier in action.

**Comment by drewclem** [comment]
@nickytonline Is there anything a11y compliant we can do about keeping the bg-image for headings on the form sections?

**Comment by nickytonline** [comment]
> @nickytonline Is there anything a11y compliant we can do about keeping the bg-image for headings on the form sections?

I'll take a peek later tonight. Didn't realize I removed the bg images, or maybe I'm misunderstanding your comment. 🙃

**Comment by drewclem** [comment]
The bg pattern is missing in your after screenshot. It may not have rendered if you'd already killed the server before snapping it?

**Comment by nickytonline** [comment]
I've updated the PR description @drewclem. It must have been some weird loading issue when I took the screenshot as it seems to be fine on my branch for the background image. Feel free to take the PR for a spin though.

**Comment by drewclem** [comment]
@nickytonline Looks good to me! I figured it was a strange loading instance. I've seen something similar before.

**Pull request merged**